src/runtime/manifest-parser.pegjs

```diff
@@ -53,6 +65,12 @@
       }
       return;
     }
+    if (result instanceof Map) {
+      for (const [key, value] of result) {
+        checkNormal(value, `${path}/${key}`);
```

Comment:
Can the key's ever contain slashes? Just wondering if this could make debugging such inputs a bit tricky.
Not sure how best to address it if so.

Reply:
I suppose they could. I could replace them with hyphens or underscores, but that could make things more confusing, depending on the exact use case.